content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -805,11 +805,14 @@ export class CollapseToggle extends React.PureComponent {
   render() {
     const {props} = this;
     const {renderAdmin} = this;
-    const action = (this.state.collapsed || !renderAdmin) ? "Expand" : "Collapse";
+    const label = `${(this.state.collapsed || !renderAdmin) ? "Expand" : "Collapse"} devtools`;
     return (<React.Fragment>
       <a href="#devtools"
-        className={`asrouter-toggle asrouter-${action.toLowerCase()}`}
-        onClick={this.renderAdmin ? this.onCollapseToggle : null}>{action} Devtools</a>
+        title={label}
+        className={`asrouter-toggle ${this.state.collapsed ? "collapsed" : "expanded"}`}
```

Comment:
Looks like state.collapsed is default false even on a plain about:newtab so it renders the className as expanded (resulting in background). Probably just needs the `|| !renderAdmin` as above.

Reply:
ah, good point